src/vse_sync_pp/analyzers/ts2phc.py

```diff
@@ -2,24 +2,14 @@
 
 """Analyze ts2phc log messages"""
 
-from .analyzer import Analyzer
+from .analyzer import TimeErrorAnalyzer
 
-class TimeErrorAnalyzer(Analyzer):
+class Ts2PhcAnalyzer(TimeErrorAnalyzer):
     """Analyze time error"""
     id_ = 'ts2phc/time-error'
     parser = id_
     def __init__(self, config):
         super().__init__(config)
-        # required system time output accuracy
-        accuracy = config.requirement('time-error-in-locked-mode/ns')
-        # limit on inaccuracy at observation point
-        limit = config.parameter('time-error-limit/%')
-        # exclusive upper bound on absolute time error for any sample
-        self._unacceptable = accuracy * limit / 100
-        # samples in the initial transient period are ignored
-        self._transient = config.parameter('transient-period/s')
-        # minimum test duration for a valid test
-        self._duration = config.parameter('min-test-duration/s')
     def prepare(self, rows):
```

Comment:
This method should be in the base TimeErrorAnalyzer.

Reply:
Fixed.